### PR TITLE
Creating the English READ.ME file

### PR DESCRIPTION
**Pull request by ArthurLins-code**
I created a English version of the READ.ME file, linking it to the Portuguese version by adding a shield right at the start of the Portuguese READ-ME file which, when clicked, redirects the user to the READ.ME_en.md file, so he/she can enjoy all the contents present in the Portuguese READ.ME, but in a language they understand it more.
This should make the package more widely accessible to non-portuguese speaking countries worldwide.

### CODE REVIEW

File: README_en.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+___Attention:___ For POF 2017-2018, Standardized Bases and Selected Expenditures are not available.
+
+## Auxiliary Programs (Dictionaries) <a name="aux"></a>
+
+Most of the package programs encounter original data stored in *.txt* format, which requires dictionaries – *.dct* format in Stata – to be read. The result is a volume of dictionaries that exceeds the 100-file limit allowed for a Stata package to be installed. Therefore, individual dictionaries are compressed into a single *.dta* file, which is read within each program. Both functions used for this are defined in the file *read\_compdct.ado*.
+
+The first program defined in this file is `write_compdct`, which can be used as follows: after running the *.ado* file to define the function, simply use the code
+
+    write_compdct, folder("/folder with dictionaries") saving("/path/dict.dta")
```
Comment (carolinamoura2000) [inline README_en.md:225]: Does this and the following commands work with the path in english or just with the path in portuguese? I  couldn't verify it in github sorry!

File: README_en.md [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+
+The function then reads all *.dct* files present in the folder and combines them into the *dict.dta* file, with each dictionary identified by a variable with its name.
+
+To transform this compressed file back into the original dictionary, we reccomend using the `read_compdct` program:
```
Comment (carolinamoura2000) [inline README_en.md:229]: To transform this compressed file back into the original dictionary, we recommend using the `read_compdct` program:

File: README_en.md [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+
+    read_compdct, compdct("dict.dta") dict_name("original_dict") out("extracted_dict.dct")
+
+which extracts the *original_dict* from the *dict.dta* file and saves it as *extracted_dict.dct*. 
+As an example, see the use of this function in the `datazoom_pnadcontinua` program:
+
+    tempfile dic // Temporary file where the extracted .dct will be saved
+
+    findfile dict.dta // Finds the dict.dta file saved by the package installation
+                      // inthe /ado/ folder and stores the path to it in the r(fn)
```
Comment (carolinamoura2000) [inline README_en.md:239]: // in the /ado/ folder and stores the path to it in the r(fn)

File: README_en.md [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+    tempfile dic // Temporary file where the extracted .dct will be saved
+
+    findfile dict.dta // Finds the dict.dta file saved by the package installation
+                      // inthe /ado/ folder and stores the path to it in the r(fn) 
+                      //macro.
+
+    read_compdct, compdct("`r(fn)'") dict_name("pnadcontinua`lang'") out("`dic'")
+      // Reads the compacted dict.dta dictionary, extracts the pnadcontinua 
+      // dictionary (or pnadcontinua_en, `lang` is empty or "_en"), and saves the 
+      // final file in the tempfile dic, which is used to read the data
```
Comment (carolinamoura2000) [inline README_en.md:245]: // final file in the tempfile dic, which is used to read the data.

**Review by carolinamoura2000 — CHANGES_REQUESTED** [review]
Here you will find my comments, that are just here to improve what was already very well-done by Arthur.

File: README_en.md [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+# DataZoom Social Stata:
+
+For the Portuguese Version, click on the Shield below:
+<!-- badges: start -->
+[![pt-br](https://img.shields.io/badge/lang-pt--br-blue.svg)](https://github.com/datazoompuc/datazoom_social_Stata/blob/English-READ.ME/README.md)
+<!-- badges: end -->
+
+<a href="https://github.com/datazoompuc/datazoom_social_Stata"><img src="https://raw.githubusercontent.com/datazoompuc/datazoom_social_stata/master/logo.jpg" align="left" width="100" hspace="10" vspace="6"></a>
+
+DataZoom Social Stata `datazoom_social` is a package that enables compatibility with microdata from surveys conducted by IBGE. With this package, it is possible to read all household surveys conducted by IBGE: Demographic Census, National Household Sample Survey (PNAD), Monthly Employment Survey (PME), Household Budget Survey (POF), and Urban Informal Economy Survey (ECINF).
```
[the remark this answers is not in the document]
Comment (ArthurLins-code) [reply README_en.md:12]: Feito

File: README_en.md (hunk 1 above)
Comment (ArthurLins-code) [reply README_en.md:225]: Carolina, I believe this is just a simple example, signalizing that the user should insert the path in both the `folder` and the `saving` commands to allow the first program to run.

**Pull request merged**